doc/releases/changelog-0.19.0.md

```diff
@@ -0,0 +1,1142 @@
+  custom quantum gradient support, and arbitrary order derivatives. This QNode is available via
+  `qml.beta.QNode`, and `@qml.beta.qnode`.
+  [(#1642)](https://github.com/PennyLaneAI/pennylane/pull/1642)
+  [(#1646)](https://github.com/PennyLaneAI/pennylane/pull/1646)
+  [(#1651)](https://github.com/PennyLaneAI/pennylane/pull/1651)
+  [(#1804)](https://github.com/PennyLaneAI/pennylane/pull/1804)
+
+  It differs from the standard QNode in several ways:
+
+  - Custom gradient transforms can be specified as the differentiation method:
+
+    ```python
+    @qml.gradients.gradient_transform
+    def my_gradient_transform(tape):
+        ...
+        return tapes, processing_fn
+
+    @qml.beta.qnode(dev, diff_method=my_gradient_transform)
+    def circuit():
+    ```
+
+  - Arbitrary :math:`n`-th order derivatives are supported on hardware using
+    gradient transforms such as the parameter-shift rule. To specify that an :math:`n`-th
+    order derivative of a QNode will be computed, the `max_diff` argument should be set.
+    By default, this is set to 1 (first-order derivatives only).
+
+  - Internally, if multiple circuits are generated for execution simultaneously, they
+    will be packaged into a single job for execution on the device. This can lead to
+    significant performance improvement when executing the QNode on remote
+    quantum hardware.
+
+  - When decomposing the circuit, the default decomposition strategy will prioritize
+    decompositions that result in the smallest number of parametrized operations
+    required to satisfy the differentiation method. Additional decompositions required
+    to satisfy the native gate set of the quantum device will be performed later, by the
+    device at execution time. While this may lead to a slight increase in classical processing,
+    it significantly reduces the number of circuit evaluations needed to compute
+    gradients of complex unitaries.
+
+  In an upcoming release, this QNode will replace the existing one. If you come across any bugs
+  while using this QNode, please let us know via a [bug
+  report](https://github.com/PennyLaneAI/pennylane/issues/new?assignees=&labels=bug+%3Abug%3A&template=bug_report.yml&title=%5BBUG%5D)
+  on our GitHub bug tracker.
+
+  Currently, this beta QNode does not support the following features:
+
+  - Non-mutability via the `mutable` keyword argument
+  - Viewing specifications with `qml.specs`
+  - The `reversible` QNode differentiation method
+  - The ability to specify a `dtype` when using PyTorch and TensorFlow.
+
+  It is also not tested with the `qml.qnn` module.
+
+* Support for differentiable execution of batches of circuits has been
+  extended to the JAX interface for scalar functions, via the beta
+  `pennylane.interfaces.batch` module.
+  [(#1634)](https://github.com/PennyLaneAI/pennylane/pull/1634)
+  [(#1685)](https://github.com/PennyLaneAI/pennylane/pull/1685)
```

Comment:
Up to you antal, but this could potentially also go in improvements! I'll leave this as your call :)